pkg/crc/machine/kubeconfig.go

```diff
@@ -186,7 +186,7 @@ func getTokenForUser(username, password, ip string, ca []byte, clusterConfig *ty
 					return nil, err
 				}
 				if strings.HasSuffix(hostname, constants.AppsDomain) {
-					port = strconv.Itoa(int(ingressHTTPSPort))
+					port = strconv.FormatUint(uint64(ingressHTTPSPort), 10)
```

Comment:
Or `fmt.Sprintf("%d", ingressHTTPSPort)`

Reply:
yeah, but this is also works so let's go with that.